pipeline.py

```diff
@@ -51,19 +61,29 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path = Path(path_to_raw_txt_data)
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        dataset = list(self.path.glob('*_raw.txt'))
+        dataset.sort(key=self._extract_file_id)
```

Comment:
To be honest, I'm not sure whether it is really vital as id of the file is either way extracted in `_extract_file_id` function and `Article` instance will get the id of the processing file from its name. In my understanding, it is just safer and more comfortable to process articles in the same order as they are in the folder as `glob()` retrieves files in another order just because they are sorted not as digits but as strings if I understood it correctly. 
Maybe I don't see something?